modules/installation-configuration-parameters.adoc

```diff
@@ -331,6 +335,10 @@ ifdef::osp[]
 |Additional security groups that are associated with control plane machines.
 |A list of one or more UUIDs as strings. For example, `7ee219f3-d2e9-48a1-96c2-e7429f1b0da7`.
 
+|`controlPlane.platform.openstack.zones`
+|OpenStack availability zones (AZs) to install nodes on. If this parameter is not set, the installer uses the default control plane zone.
```

Comment:
>  If this parameter is not set, the installer uses the default control plane zone.

I would not know what a "default control plane zone" is.

If this is not set, the Control Plane will be created:
* in the cluster-level-defined zones `platform.openstack.zones` if defined; OR
* in the default Nova zone (like in OCP v4.5)

The same applies to `compute.platform.openstack.zones` above.

Reply:
Quite helpful. Thanks!